server/mergin/sync/workspace.py

```diff
@@ -248,6 +248,10 @@ def filter_projects(
     def workspace_count():
         return 1
 
+    @staticmethod
+    def monthly_contributors_count():
+        return 0
```

Comment:
Should not be some count over project version (pushes)? Or we do not care here.

Reply:
We do not have contributors table here. But we can calculate it from project versions